sigmf-spec.md

```diff
@@ -125,44 +125,46 @@ used to describe the relationships between multiple Recordings.
 Collections and multiple Recordings can be packaged for easy storage and 
 distribution in a SigMF `Archive`.
 
+```
              ┌────────────────────┐
              │                    │
              │  SigMF Collection  │
-             │                    ├──┐
-             │   (optional file)  │  │
-             └──────────┬─────────┘  │
-                        │            │            ┌─────────────────────┐
-                        │            │  stored in │                     │
-                        │ links      ├────────────►    SigMF Archive    │
-                        │            │            │                     │
-                        │            │            │   (optional file)   │
-              ┌─────────▼─────────┐  │            └─────────────────────┘
-              │                   │  │
-              │  SigMF Recording  ├──┘
-              │                   │
-              └─────────┬─────────┘
+             │                    ├───┐
+             │   (optional file)  │   │
+             └──────────┬─────────┘   │
+                        │             │             ┌─────────────────────┐
+                        │             │  stored in  │                     │
+                        │ links       ├─────────────►    SigMF Archive    │
+                        │             │             │                     │
+                        │             │             │   (optional file)   │
+             ╔══════════▼══════════╗  │             └─────────────────────┘
+             ║                     ║  │
+             ║   SigMF Recording   ╟──┘
```

Comment:
I've got another comment later about this, but I'm not sure we're being consistent here regarding whether or not a "Recording" can actually include a virtual dataset file.

Reply:
I think what you are suggesting here is that "a sigmf recording is only a recording if it comprises both metadata and actual dataset files".

The interpretation i had used was "a recording is the fundamental unit of sigmf" and any complete data package (compliant sigmf-data + sigmf-meta, NCD sigmf-meta + other data, virtual with only sigmf-meta).

I generally agree that what this PR includes now may not be the best, and reads as possibly inconsistent.